solutions/usaco-713.mdx

```diff
@@ -2,87 +2,82 @@
 id: usaco-713
 source: USACO Bronze 2017 February
 title: Why Did the Cow Cross the Road III
-author: Óscar Garries, tickox
+author: Óscar Garries, tickox, Ryan Chou
 ---
 
-[Official Analysis](http://www.usaco.org/current/data/sol_cowqueue_bronze_feb17.html)
+[Official Analysis (Java)](http://www.usaco.org/current/data/sol_cowqueue_bronze_feb17.html)
 
-## Solution 1
+We can naively step through all of the cow entries.
+
+## Implementation
+
+**Time Complexity**: $\mathcal{O}(N)$
 
 <LanguageSection>
 <CPPSection>
 
 ```cpp
-#include <bits/stdc++.h>
+#include <iostream>
+#include <cstdio>
+#include <vector>
+#include <algorithm>
 using namespace std;
-typedef long long ll;
-typedef long double ld;
-#define f first
-#define s second
-#define len(x) (int)x.size()
-int main(){
-	ios_base::sync_with_stdio(0); cin.tie(0);
+
+int main() {
 	freopen("cowqueue.in", "r", stdin);
 	freopen("cowqueue.out", "w", stdout);
-	int n;
+	ios_base::sync_with_stdio(0); cin.tie(0);
+	
+	int n, curTime = 0; 
```

Comment:
put on separate lines, this is kinda misleading
some people might think this declares `n` as `0` as well

Reply:
godamit why did this get downvoted

Reply:
i'm not sure why people would think it declares n as 0...

Reply:
they'd read it as
"n and curTime are equal to 0, so... are they both 0?"

Reply:
i see no harm in seperating it, but also I don't think anyone would think that especially with cin >> n after.

Reply:
it's just a possible unnecessary course of confusion